src/test/java/com/thebois/models/world/structures/StructureTests.java

```diff
@@ -14,7 +15,7 @@ public void structureDoesNotEqualNull() {
         final IStructure structure = StructureFactory.createStructure(StructureType.HOUSE, 0, 0);
 
         // Act
-        final boolean isEqual = structure.equals(null);
+        final boolean isEqual = structure == null;
 
         // Assert
         assertThat(isEqual).isFalse();
```

Comment:
Here you're testing whether `createStructure` returns null or not.

So the `// Act` part is `createStructure` and the `// Assert` should be `assertThat(structure).isNotNull();`

Reply:
But actually, isn't this test supposed to test the `.equals` method? Why did you change it to `==`?